dolphinscheduler-common/pom.xml

```diff
@@ -319,6 +319,15 @@
             <artifactId>netty-all</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.squareup.okhttp3</groupId>
+            <artifactId>okhttp</artifactId>
+        </dependency>
```

Comment:
Why are you adding a duplicate dependency just the same as line 411? Can you please check whether it exists before adding one?